### DISCUSSION TITLE
Question: Namespace Level Parallelization

### DISCUSSION THREAD

**Discussion by DizzyDeveloper** [nunit/nunit#4622] [Q&A]
Is it possible to (or something to look into?) perform parallelization at a namespace level, in a similar mannerism to paralyzing at the Fixture level?
In otherwords, I have a situation where I have two groups of tests under, lets say, namespaces:

* `namespace ProjectA.Tests.Integration.GroupA;`
* `namespace ProjectA.Tests.Integration.GroupB;`

TestFixtures\Tests under a particular namespace (GroupA) cannot run in parallel with each other, because they share a resource, however, they can run in parallel with tests under a different namespace (GroupB). 
Is it possible to achieve this behaviour with the current parallelization behaviour, my understanding is no, but I thought I might see if I am missing something that might enable this behaviour.

I understand that this is a tad convoluted.

**Comment by mikkelbu** [comment]
@DizzyDeveloper I've not used the Parallelizable attribute much, but I guess that you can place the two namespaces in different assemblies and then have one attribute per assembly with `ParallelScope.Children` as selection. I don't think there is another ways with the current setup as one cannot place attributes on namespaces.
